package.json

```diff
@@ -8,11 +8,20 @@
   "engines": {
     "atom": ">=0.174.0 <2.0.0"
```

Comment:
Having the `configSchema` in `package.json` requires Atom v1.9.0 if I remember correctly.